addons/im_livechat/static/src/embed/common/chatbot/chatbot_model.js

```diff
@@ -200,6 +200,8 @@ export class Chatbot extends Record {
         );
         if (!redirectionAlreadyDone) {
             browser.location.assign(answer.redirect_link);
+        } else if (this.store.env.services.ui.isSmall) {
+            this.store.chatHub.opened[0]?.fold();
```

Comment:
Are we sure the chat hub is ready yet? We might want to add `chatHub.initPromise`. Also, we should get the chat window of the thread instead of speculating on `chatHub.opened` (probably using `.get()` + `livechatService.thread`).